challenges.yml

```diff
@@ -1699,3 +1699,448 @@ extending_smart_watcher_2:
           },
           ```
       correct: a
+
+extending_css_transitions:
+  question: |
+    Imagine I have a simple component that can show a message when one of our
+    customers have bought too much pizza.
+
+    ```vue
+    <template>
+        <div :class="{
+                [$style.component]: true,
+                [$style.tall]: showDisclaimer,
+            }"
+        >
+            <button @click="addPizzaToCart" />
+
+            <p v-show="showDisclaimer">
+                <strong>Disclaimer</strong>: Eating too much pizza can be
+                bad for your help!
```

Comment:
```suggestion
                bad for your health!
```

Reply:
Ohman! I corrected like 10 of these... It's good that you've spotted it!